README.md

```diff
@@ -7,6 +7,17 @@ A python implementation of the UA Parser (https://github.com/ua-parser, formerly
 
 ##Installing
 
+### Install via pip
+First, check that you've got the correct dependencies. For example, on Ubuntu:
+```
+$ apt-get install libyaml libyaml-dev python-dev
```

Comment:
These aren't required for pyyaml. They're just optional, so it's not correct to declare these as dependencies.